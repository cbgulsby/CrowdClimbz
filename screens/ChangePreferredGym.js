import React, { useState } from 'react';
import { 
    StyleSheet, 
    View, 
    Text,
    SafeAreaView,
    TouchableOpacity,
    Button,
    Picker,
} from 'react-native';


var value = "Gym1";

function updateGrade(val){

    value = val;
}

export default function ChangePreferredGym({navigation}){
    return(
        <SafeAreaView style={styles.container}>
            <View>
                <Text>Change Preferred Gym</Text>

                <Picker style={{borderColor: 'gray', borderWidth: 2}}
                prompt='Choose Grade'
                mode='dropdown'
                style={{height: 40, width: 350}}
                onValueChange={(itemValue, itemIndex) => updateGrade}
                >
                <Picker.Item label="Gym1" value="Gym 1" />
                <Picker.Item label="Gym2" value="Gym 2" />
                <Picker.Item label="Gym3" value="Gym 3" />
                <Picker.Item label="Gym4" value="Gym 4" />
                
                </Picker>

                <Button title="Go Back" onPress={() => navigation.navigate('Profile')} />
            </View>

        </SafeAreaView>
    );
}

const styles = StyleSheet.create({
    container: {
        paddingTop: 16,
        backgroundColor: '#4fb9ff',
        flex: 1
    }
});