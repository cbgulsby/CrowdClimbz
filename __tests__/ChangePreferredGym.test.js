import React from 'react';
import renderer from 'react-test-renderer';

import ChangePreferredGym from '../screens/ChangePreferredGym';


test('renders correctly', () => {
const tree = renderer.create(<ChangePreferredGym />).toJSON();
expect(tree).toMatchSnapshot();
});