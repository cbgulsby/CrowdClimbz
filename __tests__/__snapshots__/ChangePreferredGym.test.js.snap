// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "backgroundColor": "#4fb9ff",
      "flex": 1,
      "paddingTop": 16,
    }
  }
>
  <View>
    <Text>
      Change Preferred Gym
    </Text>
    <View
      style={
        Object {
          "height": 40,
          "width": 350,
        }
      }
    >
      <RCTPicker
        items={
          Array [
            Object {
              "label": "Gym1",
              "textColor": undefined,
              "value": "Gym 1",
            },
            Object {
              "label": "Gym2",
              "textColor": undefined,
              "value": "Gym 2",
            },
            Object {
              "label": "Gym3",
              "textColor": undefined,
              "value": "Gym 3",
            },
            Object {
              "label": "Gym4",
              "textColor": undefined,
              "value": "Gym 4",
            },
          ]
        }
        onChange={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        selectedIndex={0}
        style={
          Array [
            Object {
              "height": 216,
            },
            undefined,
          ]
        }
      />
    </View>
    <View
      accessibilityRole="button"
      accessibilityStates={Array []}
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Array [
            Object {},
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#007AFF",
                "fontSize": 18,
                "padding": 8,
                "textAlign": "center",
              },
            ]
          }
        >
          Submit Changes
        </Text>
      </View>
    </View>
    <View
      accessibilityRole="button"
      accessibilityStates={Array []}
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Array [
            Object {},
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#007AFF",
                "fontSize": 18,
                "padding": 8,
                "textAlign": "center",
              },
            ]
          }
        >
          Go Back
        </Text>
      </View>
    </View>
  </View>
</RCTSafeAreaView>
`;
